x-pack/plugins/ml/public/embeddables/anomaly_swimlane/anomaly_swimlane_embeddable_factory.tsx

```diff
@@ -127,12 +131,14 @@ export const getAnomalySwimLaneEmbeddableFactory = (
         serialize: serializeTimeRange,
       } = initializeTimeRange(state);
 
+      const swimlaneControlsDependencies$ = merge(query$, filters$, timeRange$);
```

Comment:
can you please use [fetch$](https://github.com/elastic/kibana/blob/main/packages/presentation/presentation_publishing/interfaces/fetch/fetch.ts#L122) instead? It combines all of them, also the search session id.

Reply:
Updated to use `fetch$` in: [ab9b445](https://github.com/elastic/kibana/pull/195090/commits/ab9b4454f64310a10032a04b4721d472e0317a3a)